### ISSUE TITLE
Linux [ubuntu] install issues

### ISSUE THREAD

**Issue by cristi-zz** [GuidoBartoli/sherloq#14]
Hi! 

I am trying to install on Ubuntu 16.04 your library.

I met the following error while installing requirements.txt:

```
ERROR: tensorflow 2.2.0 has requirement scipy==1.4.1; python_version >= "3", but you'll have scipy 1.5.1 which is incompatible.
```

Ok, ignored, moving forward.

When launching, I get:

```
python sherloq.py
WARNING:tensorflow:From /home/*****/miniconda3/envs/serloq/lib/python3.8/site-packages/tensorflow/python/compat/v2_compat.py:96: disable_resource_variables (from tensorflow.python.ops.variable_scope) is deprecated and will be removed in a future version.
Instructions for updating:
non-resource variables are not supported in the long term
qt.qpa.plugin: Could not load the Qt platform plugin "xcb" in "" even though it was found.
This application failed to start because no Qt platform plugin could be initialized. Reinstalling the application may fix this problem.

Available platform plugins are: eglfs, linuxfb, minimal, minimalegl, offscreen, vnc, wayland-egl, wayland, wayland-xcomposite-egl, wayland-xcomposite-glx, webgl, xcb.

[1]    5775 abort (core dumped)  python sherloq.py
```

[I am using anaconda instead of virtualenv, but the starting env was empty (eg python 3.8 only) ]

Tried to fiddle with opencv (headless, contrib) and pyqt5. The 2nd problem still persists.

**LE:** I ran ``sudo apt-get install libxcb-xinerama0`` as found here: https://forum.qt.io/topic/93247/qt-qpa-plugin-could-not-load-the-qt-platform-plugin-xcb-in-even-though-it-was-found

**Comment by cristi-zz** [comment]
To help others, this is roughly what I did on my Ubuntu 16.04:

1) Download and install Anaconda: https://www.anaconda.com/products/individual 
There are install instructions (GUI, non GUI, etc)

One can also install miniconda [no GUI but is smaller]

2) Create a new python environment:

2.1) Open a console. On Windows one must start a Conda Console [or sth] from the Start menu. Don't know on Mac.
Run: ``conda create --copy -n serloq python``
Accept Yes when it prompts

After install ends, type in the same console ``conda activate serloq`` to activate the env. Don't close the console.
Depending on OS there should be a ``(serloq)`` prepended to the console prompt. 

2.2) Download and unpack the repo from the GitHub page.

2.3) Go to ``sherloq/gui`` folder and run ``pip install -r requirements.txt`` MAKE SURE THAT THE CONDA ENV IS STILL ON!!!

2.3.1) After trial and error I had to install ``sudo apt-get install libxcb-xinerama0``

2.4). Launch the application:  ``python sherloq.py``

A GUI is opened. I DID NOT clicked around, to see if everything works.

Hope it helps!

**Comment by GuidoBartoli** [comment]
I do not use Conda framework, so I did not check how to install the software in that scenario, many thanks! :+1: 
I updated the README with your instructions.

**Comment by GuidoBartoli** [comment]
> I am trying to install on Ubuntu 16.04 your library.
> 
> I met the following error while installing requirements.txt:
> 
> ```
> ERROR: tensorflow 2.2.0 has requirement scipy==1.4.1; python_version >= "3", but you'll have scipy 1.5.1 which is incompatible.
> ```

I found that it is caused by the order in which packages appear inside `requirements.txt'. I used `pip-chill' to clean the list and it produced a sorted list where `tensorflow` was at the bottom and some other package installed `scipy 1.5.1` before it.
This is a known limitation of `pip`, but I am trying different combinations to find the one without conflict.
Unfortunately `tensorflow` is the most "squeamish" packages, because the others can happily live together all at the latest versions without complaining... :disappointed:
